content/ch-quantum-hardware/ac-Stark-shift.ipynb

```diff
@@ -46,10 +46,12 @@
     "# Importing standard Qiskit libraries and configuring account\n",
```

Comment:
<p>a the control channel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Here, I intended to describe the explanation of conn_chan used in the experiment.</p>

Reply:
I will change from "the control channel" to "a control channel".